### DISCUSSION TITLE
Flight model simulation

### DISCUSSION THREAD

**Discussion by memy009** [JSBSim-Team/jsbsim#858] [Support requests]
Hello,
So glad to know that a team can potentially help me on my project :) I am currently working on the flight transtion of a VTOL, so currently I'm in the modeling and control part and this modeling must be tested on jsbsim. I have to simulate my model on jsbsim and I don't know where to start, can you help me in that because I am really really lost.
Thank U.

**Comment by seanmcleod** [comment]
Well tell us what information and data you have, and in what particular formats etc. and then we can give you some initial pointers. The more information you can give us the better we'd be able to give you helpful pointers.

**Comment by memy009** [comment]
I would like to do two things: the first is to know how to simulate an existing flight model on JSBSim to see how it works, the second, I just have a simplified model that I have to simulate the performances like the pitch variation, speed ... etc. I need simple code, preferably in python that does this. Like I said, I've never used the tool so, I don't really know where to start. Maybe you can start by telling me what I can do on JSBSim.

**Comment by seanmcleod** [comment]
Have you read through the JSBSim Reference Manual linked to at - https://github.com/JSBSim-Team/jsbsim#user-documentation

**Comment by memy009** [comment]
It's after reading it that I've asked the question.

**Comment by memy009** [comment]
thank U anyways

**Comment by seanmcleod** [comment]
> I just have a simplified model that I have to simulate the performances like the pitch variation, speed ... etc.

Have you tried out the script support as mentioned in the reference manual and by looking at some of the sample scripts in the JSBSim repo? For example you can use a script to trim an aircraft model, and then use script entries to change some of the controls, e.g. to perform a pitch step input and use the output logging facilities to record states like pitch angle, speed etc. and then plot those to see how the aircraft responds to the inputs you supplied.

The other option as you mentioned is via the Python API support, take a look at the following discussion in which I posted some python example code on how to trim the aircraft and then also apply a rudder kick, record some data and plot it.

[JSBSim sideslip angle, rudder kick](https://github.com/JSBSim-Team/jsbsim/discussions/519)

**Comment by ingStudent** [comment]
Hi memmy 009, 

I'm working on a similar subject, could you tell me in which part of jsbsim you are maybe we could help each other? 

Thank U.
